Compile warning with missing-field-initializers

**Describe the bug**

I'm trying to integrate `tree-sitter-hcl` with [difftastic](https://github.com/Wilfred/difftastic)

But meeting the following compile warning:

```shell
warning: vendor/tree-sitter-hcl-src/scanner.cc: In member function ‘bool {anonymous}::Scanner::scan(TSLexer*, const bool*)’:
warning: vendor/tree-sitter-hcl-src/scanner.cc:99:39: warning: missing initializer for member ‘{anonymous}::Context::heredoc_identifier’ [-Wmissing-field-initializers]
warning:    99 |       Context ctx = { QUOTED_TEMPLATE };
warning:       |                                       ^
warning: vendor/tree-sitter-hcl-src/scanner.cc:117:48: warning: missing initializer for member ‘{anonymous}::Context::heredoc_identifier’ [-Wmissing-field-initializers]
warning:   117 |         Context ctx = { TEMPLATE_INTERPOLATION };
warning:       |                                                ^
warning: vendor/tree-sitter-hcl-src/scanner.cc:145:44: warning: missing initializer for member ‘{anonymous}::Context::heredoc_identifier’ [-Wmissing-field-initializers]
warning:   145 |         Context ctx = { TEMPLATE_DIRECTIVE };
warning:       |                                            ^
```

**To Reproduce**
Steps to reproduce the behavior:
1. Build with clang

**Expected behavior**
No warning

**Screenshots**
![image](https://user-images.githubusercontent.com/5351546/164759701-c80ec380-4b26-4ae4-9ca9-fb9f19efa293.png)

---

I'm glad to send a PR if fixing the warning looks good to you :smile:

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -19,4 +19,4 @@ jobs:
         uses: egor-tensin/setup-mingw@v2
 
       - name: build
-        run: ${{ matrix.compiler }} -o scanner.o -I./src -c src/scanner.cc
+        run: ${{ matrix.compiler }} -o scanner.o -I./src -c src/scanner.cc -Werror

--- a/src/scanner.cc
+++ b/src/scanner.cc
@@ -96,7 +96,7 @@ public:
     }
     // manage quoted context
     if (valid_symbols[QUOTED_TEMPLATE_START] && !in_quoted_context() && lexer->lookahead == '"') {
-      Context ctx = { QUOTED_TEMPLATE };
+      Context ctx = { QUOTED_TEMPLATE, "" };
       context_stack.push_back(ctx);
       return accept_and_advance(lexer, QUOTED_TEMPLATE_START);
     }
@@ -114,7 +114,7 @@ public:
     ) {
       advance(lexer);
       if (lexer->lookahead == '{') {
-        Context ctx = { TEMPLATE_INTERPOLATION };
+        Context ctx = { TEMPLATE_INTERPOLATION, "" };
         context_stack.push_back(ctx);
         return accept_and_advance(lexer, TEMPLATE_INTERPOLATION_START);
       }
@@ -142,7 +142,7 @@ public:
     ) {
       advance(lexer);
       if (lexer->lookahead == '{') {
-        Context ctx = { TEMPLATE_DIRECTIVE };
+        Context ctx = { TEMPLATE_DIRECTIVE, "" };
         context_stack.push_back(ctx);
         return accept_and_advance(lexer, TEMPLATE_DIRECTIVE_START);
       }

